src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/CategoryCard/components/CategoryCard/CategoryCard.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { CategoryCustomData } from '@catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/utils/types';
+import {
+  findInstanceType,
+  seriesDetails,
+} from '@catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/utils/utils';
+import { AngleDownIcon } from '@patternfly/react-icons';
+
+import './CategoryCard.scss';
+
+type InstanceTypeCardProps = {
+  toggleRef?: React.Ref<HTMLDivElement>;
+  isExpanded?: boolean;
+  onClick: (...args: any[]) => void;
+  onFocus: (...args: any[]) => void;
+  onMouseOver: (...args: any[]) => void;
+  customData: CategoryCustomData;
+};
+
+const CategoryCard: React.FC<InstanceTypeCardProps> = ({
+  toggleRef,
+  isExpanded,
+  onClick,
+  customData,
+}) => {
+  const { category, selectedCategory, selectedSize } = customData;
+  const details = seriesDetails[category];
+  const { Icon, instanceTypes, seriesLabel, title } = details;
+
+  const isSelectedCategory = selectedCategory === category;
+  const displaySelection = isSelectedCategory && Boolean(selectedSize);
+  const selectedInstanceTypeDetails = findInstanceType(selectedSize, instanceTypes);
+
+  const handleClick = () => {
+    onClick();
+  };
+
+  return (
+    <div
+      ref={toggleRef}
+      className={classNames('category-card', { 'selected-category': isSelectedCategory })}
+      onClick={handleClick}
+      {...(isExpanded && { 'aria-expanded': true })}
+    >
+      <div className="category-card__icon">{Icon && <Icon />}</div>
```

Comment:
<div className="category-card__icon">{Icon && <Icon />}</div> in this case we will have an empty div

 {Icon && <div className="category-card__icon"><Icon /></div>} can work also?

Reply:
Fixed!